CalculatorWithTests/Calculator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CalculatorWithTests
+{
+    public class Calculator
+    {
+        public int Add(int number1, int number2)
+        {
+            throw new NotImplementedException();
+        }
+
+        public double Subtract(double minuend, double subtrahend)
+        {
+            throw new NotImplementedException();
+        }
+
+        public long Multiply(long factor1, long factor2)
+        {
+            throw new NotImplementedException();
+        }
+
+        public double Divide(double num1, double num2)
+        {
+            throw new NotImplementedException();
+        }
+
+        public long Factorial(double recursive, double recursive2)
```

Comment:
I'm not sure what the second parameter would be for a Factorial method, and the parameter names don't help me find out.  This may be due to Factorial being a new concept, but there's only one number that needs to be passed in